src/system-core.js

```diff
@@ -29,14 +29,14 @@ const systemJSPrototype = SystemJS.prototype;
 
 systemJSPrototype.prepareImport = function () {};
 
-systemJSPrototype.import = function (id, parentUrl) {
+systemJSPrototype.import = function (id, parentUrl, importerSetters) {
```

Comment:
Again, I would prefer not to overload the `import` function this way, but work out a way to handle these primitives in the registry API.

See the suggestion above re having delete return a function.